freqtrade/data/dataprovider.py

```diff
@@ -311,11 +311,14 @@ def historic_ohlcv(
             timerange = TimeRange.parse_timerange(None if self._config.get(
                 'timerange') is None else str(self._config.get('timerange')))
 
-            # It is not necessary to add the training candles, as they
-            # were already added at the beginning of the backtest.
-            startup_candles = self.get_required_startup(str(timeframe), False)
+            startup_candles = self.get_required_startup(str(timeframe))
             tf_seconds = timeframe_to_seconds(str(timeframe))
             timerange.subtract_start(tf_seconds * startup_candles)
+
+            logger.info(f"Loading data for {pair} {timeframe} "
```

Comment:
with the assumption that this is going to run multiple times for a backtest / pair (not sure how often for now) , i'm not sure it's a good idea (at least not at info level).

a log at info level always prints to screen - which is a (although quick if treated as "one entry") delay.

so for something that's expected to run quickly, printing a lot to screen can delay the whole function significantly ...

Reply:
I understand your concern. I thought the additional info log might provide more meaningful information compared to the log inside `get_required_startup`, which currently prints `Increasing startup_candle_count for freqai to .*` multiple times at the info level. Upon closer observation, I noticed that this log message is printed for each pair and timeframe combination used in the strategy. This occurs once before the first training session begins.

To ensure the changes align with your preferences, could you please let me know which option you prefer:

1.  Remove the new log message or adjust its verbosity to "debug."
2.  Consider changing the verbosity of the log message right before it to "debug" or leaving it as is.

I appreciate your time.

Reply:
i don't have a strong preference at the moment.
I don't have the loading mechanics for freqAI 100% clear without doing some tests - if this will only appear before the first training, there's no harm in leaving it as you put it.

As a side note: the `Increasing startup_candle_count for freqai to ` message might benefit from adding the timeframe ... so it's at least not 20x EXACTLY the same . . . 😆

Reply:
> As a side note: the `Increasing startup_candle_count for freqai to ` message might benefit from adding the timeframe ... so it's at least not 20x EXACTLY the same . . . 😆

I've added that.